nixos/modules/services/misc/gitlab.nix

```diff
@@ -1145,7 +1251,11 @@ in {
         TimeoutSec = "infinity";
         Restart = "on-failure";
         WorkingDirectory = "${cfg.packages.gitlab}/share/gitlab";
-        ExecStart = "${cfg.packages.gitlab.rubyEnv}/bin/puma -C ${cfg.statePath}/config/puma.rb -e production";
+        ExecStart = "${cfg.packages.gitlab.rubyEnv}/bin/puma "
+                    + "-e production "
+                    + "-C ${cfg.statePath}/config/puma.rb "
+                    + "-w ${cfg.puma.workers} "
+                    + "-t ${cfg.puma.threadsMin}:${cfg.puma.threadsMax} ";
```

Comment:
```suggestion
        ExecStart = "${cfg.packages.gitlab.rubyEnv}/bin/puma "
          + "-e production "
          + "-C ${cfg.statePath}/config/puma.rb "
          + "-w ${cfg.puma.workers} "
          + "-t ${cfg.puma.threadsMin}:${cfg.puma.threadsMax} ";
```

Reply:
Hm, I find the original more readable than your suggestion. Alternatively, we could use `concatStringsSep " "` instead. That's probably better anyway, to avoid potential future problems.

Reply:
Aligning things like this creates extra whitespace updates when the first line changes and are not formattable by nixpkgs-fmt.

Reply:
Only if the attribute name assigned to changes, which isn't common enough to worry about and not worth sacrificing readability over. The original formatting was done by `nix-mode`'s SMIE indenter, which often indents code more naturally than `nixpkgs-fmt` imo. Either way, it's replaced by `concatStringsSep " "` now.

Reply:
> nix-mode's SMIE indenter

Thats emacs magic, right? So it is not accessible to a major portion of users.

Reply:
Yes, point being the indentation wasn't completely arbitrary - it's the way my editor does it and I much prefer it to the `nixpkgs-fmt` indentation.